src/seedu/addressbook/commands/employee/ClockIn.java

```diff
@@ -0,0 +1,62 @@
+package seedu.addressbook.commands.employee;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Set;
+
+import seedu.addressbook.commands.Command;
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.employee.Attendance;
+import seedu.addressbook.data.employee.EmployeeName;
+import seedu.addressbook.data.employee.Timing;
+
+/**
+ * Clocks in for the specified employee based on the current time.
+ */
+public class ClockIn extends Command {
```

Comment:
I think this class should be named `EmployeeClockInCommand` for consistency with the other class name

Reply:
Or it can be named `AttendaceClockOutCommand`